styles/footer.css

```diff
@@ -117,6 +120,26 @@ footer .copyright {
 }
 
 
```

Comment:
You don't have to add this media query. It is making the footer items center on smaller displays. 
There is already a media query for smaller screens.